internal/server/kong/ws/node.go

```diff
@@ -185,6 +187,12 @@ func (n *Node) readThread() error {
 
 func (n *Node) sendConfig(ctx context.Context, payload *Payload) error {
 	content, err := n.getPayload(ctx, payload)
+	metrics.Histogram("data_plane_config_size",
+		float64(unsafe.Sizeof(content)),
```

Comment:
This won't do what you expect, as it's grabbing the size of the entire `config.Content` object & not the size of the actual config.

Let's be sure we're capturing the size of the JSON payload being sent to the data plane, and ideally, it's uncompressed size as that's what matters. Let's modify the `config.Content` struct to include the size in bytes as well, so that we can log it here.

Reply:
I will address this as a separate issue.

Reply:
That works with me, thanks for removing this metric then.